pkg/asset/quota/aws/aws.go

```diff
@@ -137,7 +138,12 @@ func machineTypeToQuota(t string, instanceTypes map[string]InstanceTypeInfo) quo
 	switch class {
 	case "a", "c", "d", "h", "i", "m", "r", "t", "z":
 		return quota.Constraint{Name: "ec2/L-1216C47A", Count: info.vCPU}
+	case "g":
+		return quota.Constraint{Name: "ec2/L-3819A6DF", Count: info.vCPU}
 	default:
+		if strings.HasPrefix(strings.ToLower(t), "vt") {
+			return quota.Constraint{Name: "ec2/L-3819A6DF", Count: info.vCPU}
+		}
 		return quota.Constraint{Name: "ec2/L-7295265B", Count: info.vCPU}
```

Comment:
Let's add a case for "X" class explicitly rather than defaulting to "X". If the class type is one that the installer does not know, then we should warn the user that we are not checking that quota and return a constraint with a 0 count. That way we are protected from new classes being added in the future that we would otherwise assume are "X".